esp/scm/ws_decoupledlogging.ecm

```diff
@@ -17,37 +17,63 @@
 
 EspInclude(common);
 
+ESPStruct [nil_remove] LogAgentGroup
+{
+    string GroupName;
+    ESParray<string> AgentNames;
+};
+
+ESPStruct [nil_remove] LogAgentStatus
+{
+    string AgentName;
+    string Status;
+};
+
+ESPStruct [nil_remove] LogAgentGroupStatus
+{
+    string GroupName;
+    string GroupStatus;
+    ESParray<ESPstruct LogAgentStatus> LogAgentStatus;
```

Comment:
There are a couple naming inconsistencies in this file. This value name is singular when other arrays are plural. "GroupName" omits a "LogAgent" prefix, but this has a "Log" prefix ("Agent" is necessary in this case).